pages/common/git-push.md

```diff
@@ -1,23 +1,31 @@
 # git push
 
-> Push commits to a remote repository.
+> Push commits to a remote (upstream) repository.
```

Comment:
"upstream" wasn't used here as the concrete name of a remote (we could call them "origin", "upstream", "remote" or anything else if we wanted), but rather as a commonly used synonym -- so @agnivade's concern is valid.

My addition of the term here is because it is often used in that sense (including in several tldr pages). But we probably should avoid promoting that equivalence, since the terms "upstream" and "downstream" have specific meanings in the context of software dependency, which don't necessarily map with the meanings of "local" and "remote" in the context of computer networks in general (and distributed networks of git repositories in particular).

If you agree, I'll make the change.